### DISCUSSION TITLE
posts/2021/11/making-tailwind-jit-work-hugo/

### DISCUSSION THREAD

**Discussion by giscus** [brycewray/comments#6] [Announcements]
# posts/2021/11/making-tailwind-jit-work-hugo/

What to do until Tailwind CSS’s JIT mode works OoTB with Hugo.

https://www.brycewray.com/posts/2021/11/making-tailwind-jit-work-hugo/

**Comment by darrenaddy** [comment]
I swear, every time I learn something new and Google to learn more - I find an article written by you. Great work! I'd love to be able to stay on top of everything like you do, but at least I can FOLLOW your writings. :) Thank you!
PS . . . I hope you will continue to update on this (or write a new article on current Best Practices for those who wish to learn to use Tailwind 3 JIT with Hugo).

**Reply by brycewray** [reply]
Thank you for the kind words! I actually do have such an article: https://www.brycewray.com/posts/2022/03/making-tailwind-jit-work-hugo-version-3-edition/ — although, please note, it’s still a workaround; as yet, there still is no official Hugo way of doing Tailwind/JIT.

**Comment by darrenaddy** [comment]
As a separate issue... is there a way to PRINT your pages and not lose content behind your top floating menu bar??
   If I try to use a custom margin, it just pushes the floating menu bar (and the content hiding behind it) down, so that doesn't work. I find this issue on a lot of Hugo sites that use the floating menu bar and would love to know how to code this away for my own Hugo sites. Thx!

**Reply by brycewray** [reply]
Yes, I probably need to add some CSS to handle that. My apologies for your inconvenience in the interim!

**Edit, 2022-04-21**: Have now done so. Hope it works better for you.
